components/compositing/compositor.rs

```diff
@@ -642,6 +662,10 @@ impl<Window: WindowMethods> IOCompositor<Window> {
                 }
             }
 
+            (Msg::Dispatch(func), _) => {
+                func();
```

Comment:
I think that might be useful in other future cases. The comments already say what it is used for now, but I'd bet that other API will require it too.